.circleci/config.yml

```diff
@@ -39,6 +35,36 @@ jobs:
           command: |
               ~/.local/bin/gitlint --commits origin/master..HEAD
 
+  # Check that the CHANGELOG has been updated in the current branch
+  check-changelog:
+    docker:
+      - image: circleci/buildpack-deps:stretch-scm
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - run:
+          name: Check that the CHANGELOG has been modified in the current branch
+          command: |
+            git whatchanged --name-only --pretty="" origin..HEAD | grep CHANGELOG
+
+  # Check lockfile consistency.
+  #
+  # Make sure that the package-lock.json file has been updated consequently to
+  # a package.json file.
+  check-lockfile:
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    working_directory: ~/etherpad
+    steps:
+      # Checkout repository sources
+      - checkout
+      - run:
+          name: Check lockfile consistency
+          command: |
+            if git whatchanged --name-only --pretty="" origin..HEAD | grep package.json; then
```

Comment:
IMO this is wrong, if you change something else than `dependencies` or `devDependencies` the lock will not be updated and the CI will fail.

Reply:
Awwwh. You are totally right. Even if in 95% cases, we will only change/update (dev)dependencies...

I hesitated while writing this part, but I think the best way to ensure both `package.json` and related lock file are synchronous is to switch to Yarn and use the `--frozen-lockfile` mode. I'll handle it in a separate PR if you don't mind?

Reply:
Ok but you remove this check in this PR ?

Reply:
I would prefer having false positives than to check at all. I'll make sure this job is not required.